src/blockchain/log-processors/market-created.ts

```diff
@@ -100,6 +105,9 @@ export function processMarketCreatedLog(db: Knex, augur: Augur, log: FormattedEv
             (next: AsyncCallback): void => {
               db.insert(marketsDataToInsert).into("markets").asCallback(next);
             },
+            (next: AsyncCallback): void => {
+              db.raw("insert into search_en(marketId, content) values( ?, ? )", [fullTextStringInsert.marketId, fullTextStringInsert.content]).asCallback(next);
```

Comment:
Does the normal db.insert({}) method not work for that virtual table?

Reply:
I’d need to try that. I went with raw b/c it worked